### ISSUE TITLE
Subtitles get out of sync when resuming a movie, or the audio/video is out of sync with the subtitles.

### ISSUE THREAD

**Issue by joost00719** [jellyfin/jellyfin-androidtv#3492]
### Describe the bug

Reproduce steps:

1. Watch a movie until like 50% of it.
2. Notice that the subtitles are (probably) in sync.
3. Shut down jellyfin app.
4. Start Jellyfin again, and try to resume the movie.
5. Notice that the subtitles are out of sync. However this doesn't happen at every movie.

I also noticed something else odd, which could explain why this bug happens.

If I go to timestamp 1:00:00 on the androidtv client (chromecast), and I go to 1:00:00 on the Windows 10 client, it seems like the androidtv client is like 5-10 seconds behind the Windows 10 client. The subtitles are 100% exactly on the same timestamp, so those appear at the identical time if you sync the two clients up, but the video and audio on the androidtv client seems to be 5-10 seconds behind, which could explain the subtitle sync issue.
Long story short, it looks like the audio/video get's out of sync with the time, but the subtitles are actually correct.

For me this is a big issue since I have trouble hearing voices with background noise, especially if it's not my native tongue.

If any more info is needed, I am able to (privately) provide a test-account to my JF server to a developer, with sftp access to my library.

### Logs

```
[2024-04-08 13:03:05.430 +02:00] [INF] [74] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "joost". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2024-04-08 13:03:05.431 +02:00] [INF] [74] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="AndroidTV-libVLC", Path="/data/media/movies/Harry Potter and the Deathly Hallows Part 2 (2011)/Harry.Potter.and.the.Deathly.Hallows.Part.2.2011.1080p.10bit.BluRay.6CH.x265.HEVC-PSA.mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/7a35d69c-9514-7a12-6085-859f3e9f0d8b/stream.mkv?MediaSourceId=7a35d69c95147a126085859f3e9f0d8b&Static=true&VideoCodec=hevc&AudioCodec=aac&AudioStreamIndex=2&api_key=<token>&Tag=bca4acb4a71f9ec7e60db48d1994d1f6"
[2024-04-08 13:03:05.581 +02:00] [INF] [36] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "joost". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2024-04-08 13:03:05.581 +02:00] [INF] [36] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="AndroidTV-ExoPlayer", Path="/data/media/movies/Harry Potter and the Deathly Hallows Part 2 (2011)/Harry.Potter.and.the.Deathly.Hallows.Part.2.2011.1080p.10bit.BluRay.6CH.x265.HEVC-PSA.mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=Transcode, TranscodeReason=AudioChannelsNotSupported ) "media:/videos/7a35d69c-9514-7a12-6085-859f3e9f0d8b/master.m3u8?MediaSourceId=7a35d69c95147a126085859f3e9f0d8b&VideoCodec=hevc,h264,hevc&AudioCodec=aac&AudioStreamIndex=2&VideoBitrate=99616000&AudioBitrate=384000&AudioSampleRate=48000&MaxFramerate=23.976025&api_key=<token>&RequireAvc=false&Tag=bca4acb4a71f9ec7e60db48d1994d1f6&SegmentContainer=ts&BreakOnNonKeyFrames=False&hevc-level=120&hevc-videobitdepth=10&hevc-profile=main10&hevc-audiochannels=6&aac-profile=he-aac&aac-audiochannels=2&TranscodeReasons=AudioChannelsNotSupported"
[2024-04-08 13:03:05.955 +02:00] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Adding playback tracker : 2e352282e0d23b46b66a01bd96896342f814fbbe-342c3b788c2e49dfb6226644434d66ef-7a35d69c95147a126085859f3e9f0d8b
[2024-04-08 13:03:05.955 +02:00] [INF] [36] Jellyfin.Plugin.PlaybackReporting.Data.PlaybackTracker: PlaybackTracker : Adding Start Event : 04/08/2024 13:03:05
[2024-04-08 13:03:05.955 +02:00] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Creating StartPlaybackTimer Task
[2024-04-08 13:03:05.955 +02:00] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : Entered
[2024-04-08 13:03:06.836 +02:00] [INF] [73] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : "2e352282e0d23b46b66a01bd96896342f814fbbe-342c3b788c2e49dfb6226644434d66ef-7a35d69c95147a126085859f3e9f0d8b"
[2024-04-08 13:03:07.946 +02:00] [INF] [56] Jellyfin.Api.Helpers.TranscodingJobHelper: "/usr/lib/jellyfin-ffmpeg/ffmpeg" "-analyzeduration 200M -fflags +genpts  -i file:\"/data/media/movies/Harry Potter and the Deathly Hallows Part 2 (2011)/Harry.Potter.and.the.Deathly.Hallows.Part.2.2011.1080p.10bit.BluRay.6CH.x265.HEVC-PSA.mkv\" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 0:1 -map -0:s -codec:v:0 copy -tag:v:0 hvc1 -bsf:v hevc_mp4toannexb -start_at_zero -codec:a:0 libfdk_aac -ac 2 -ab 384000 -ar 48000 -af \"volume=2\" -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 6 -hls_segment_type mpegts -start_number 0 -hls_segment_filename \"/config/data/transcodes/b0c2898c3a5c96e5cb336ce3ed084331%d.ts\" -hls_playlist_type vod -hls_list_size 0 -y \"/config/data/transcodes/b0c2898c3a5c96e5cb336ce3ed084331.m3u8\""
[2024-04-08 13:03:09.122 +02:00] [INF] [69] Jellyfin.Api.Helpers.TranscodingJobHelper: Stopping ffmpeg process with q command for "/config/data/transcodes/b0c2898c3a5c96e5cb336ce3ed084331.m3u8"
[2024-04-08 13:03:09.189 +02:00] [INF] [69] Jellyfin.Api.Helpers.TranscodingJobHelper: FFmpeg exited with code 0
[2024-04-08 13:03:09.190 +02:00] [INF] [69] Jellyfin.Api.Helpers.TranscodingJobHelper: "/usr/lib/jellyfin-ffmpeg/ffmpeg" "-analyzeduration 200M -ss 01:18:16.567 -fflags +genpts  -i file:\"/data/media/movies/Harry Potter and the Deathly Hallows Part 2 (2011)/Harry.Potter.and.the.Deathly.Hallows.Part.2.2011.1080p.10bit.BluRay.6CH.x265.HEVC-PSA.mkv\" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 0:1 -map -0:s -codec:v:0 copy -tag:v:0 hvc1 -bsf:v hevc_mp4toannexb -start_at_zero -codec:a:0 libfdk_aac -ac 2 -ab 384000 -ar 48000 -af \"volume=2\" -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 6 -hls_segment_type mpegts -start_number 782 -hls_segment_filename \"/config/data/transcodes/b0c2898c3a5c96e5cb336ce3ed084331%d.ts\" -hls_playlist_type vod -hls_list_size 0 -y \"/config/data/transcodes/b0c2898c3a5c96e5cb336ce3ed084331.m3u8\""
[2024-04-08 13:03:25.954 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: session.RemoteEndPoint : "192.168.1.254"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : event_playing_id     = "7a35d69c95147a126085859f3e9f0d8b"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : event_user_id        = "342c3b788c2e49dfb6226644434d66ef"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : event_user_id_int    = 1
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : session_playing_id   = "7a35d69c95147a126085859f3e9f0d8b"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : session_user_id      = "342c3b788c2e49dfb6226644434d66ef"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : play_method          = "Transcode (v:direct a:aac)"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : e.ClientName         = "Android TV"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : e.DeviceName         = "Joost"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : ItemName             = "Harry Potter and the Deathly Hallows: Part 2"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : ItemId               = "7a35d69c95147a126085859f3e9f0d8b"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : ItemType             = "Movie"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : All matches, playback registered
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Playback tracker found, adding playback info : "2e352282e0d23b46b66a01bd96896342f814fbbe-342c3b788c2e49dfb6226644434d66ef-7a35d69c95147a126085859f3e9f0d8b"
[2024-04-08 13:03:25.955 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Saving playback tracking activity in DB
[2024-04-08 13:03:25.977 +02:00] [INF] [65] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: StartPlaybackTimer : Exited
[2024-04-08 13:03:27.078 +02:00] [INF] [36] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : "2e352282e0d23b46b66a01bd96896342f814fbbe-342c3b788c2e49dfb6226644434d66ef-7a35d69c95147a126085859f3e9f0d8b"
[2024-04-08 13:03:49.305 +02:00] [INF] [73] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : "2e352282e0d23b46b66a01bd96896342f814fbbe-342c3b788c2e49dfb6226644434d66ef-7a35d69c95147a126085859f3e9f0d8b"
```

### Application version

jellfyin-androidtv 0.16.7 release

### Where did you install the app from?

Google Play

### Device information

Chromecast HD

### Android version

Android TV 12 (STTL.240206.002)

### Jellyfin server version

10.8.13 (docker)

**Comment by joost00719** [comment]
Iirc it was a direct stream. But I'll be double checking later today and report back.
________________________________

**Comment by joost00719** [comment]
Sorry, something came up yesterday.
Anyways, it isn't being transcoded:
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/120cc56d-569b-451a-9b86-294c11649339)


However, I tried to force transcoding by putting the quality to 1Mbit/s, and this "fixes" the sync issues.
"fixes" in quotes because it makes the movie look bad obviously.
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/a90d5616-4915-43cc-a6d7-b30411b4cabb)


Is there any other way to force encoding, so I can temp-fix this until a proper fix is out?

**Comment by joost00719** [comment]
Hi VampiricAlien,

Thanks for the suggestion, however it did not temp-fix my issue.
Are there other ways to force transcoding without losing resolution / on higher bitrates?

In case it is important, my setup is like this:
i5 9500T running Proxmox
Proxmox has a Debian VM with iGPU pass-through
Debian vm runs Docker with GPU device mapping
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/599e51f6-b6a1-4daa-9437-5a2e0ee036ec)


And this are my JF settings for hardware decoding
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/8cd436bc-fc24-454f-9460-ca29d050d3f6)

**Comment by joost00719** [comment]
I've set it manually to 120mbps, but it's still direct stream and out of sync
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/24794dec-c53d-41a0-8dcb-46dc322aab48)
![IMG_20240415_213117](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/334aaba5-6106-462c-aeb2-8c785d44127f)

I didn't know how to get that video detail in the format how you have it, I hope this screenshot is enough:
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/3e1618de-cd3b-46cf-bcf1-cb348f1b0f6d)
If it's not, please let me know how I can get more information.

**Comment by joost00719** [comment]
Yes, on my phone and on my pc using the JellyFin windows client the subtitles are correct.

**Comment by joost00719** [comment]
I haven't seen that option anywhere yet, but I'll see if I can find it tomorrow, and report back. Thanks for the suggestion.

**Comment by joost00719** [comment]
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/2d4baf9c-e10d-459e-9d79-191eab1ceb07)
I just enabled it, rebooted JF and tried it again without luck.
I think what I said in the original post is what is actually happening, which is the video/audio getting out of sync, but the subtitles actually being "correct".

windows:
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/235f30b5-6331-41a4-999c-63490978b5bc)
Chromecast:
![IMG_20240416_183904](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/d6459558-cb67-4df4-ae85-ffb4105ea531)

Notice that both frames are at 35:30, have a different scene, but the exact same subtitles. This confirms for me that the subtitles are actually correct, but the video itself is somewhere else than JF expects, resulting in a sync issue between video and subtitles.

**Comment by joost00719** [comment]
I'll be waiting on a fix for now. And in the mean time I'll use a nuc to watch movies from the TV.

Thanks for helping and let me know if I can help in any way (e.g. Test a beta version)

**Comment by Deagarys** [comment]
I've been having this issue aswell, only happens on androidTv client. Subtitles that burn in are still correct even after resuming the movie/serie.

**Comment by hcharbonnier** [comment]
In the Android App (or Web client), in the Subtiltes setting menu, you have an option to force burning all the subtitles in the video.
Then restart the AndroidTV app and all your subtiltes will be burned in the video via transcoding.
It is a pity this setting is not directly avilable in the AndroidTV app...

**Comment by hcharbonnier** [comment]
It is not "Subtitle mode"but the parameter right after. I don't know it's name in english but it should something like "Burn the subtitles"

**Comment by joost00719** [comment]
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/78ba8492-cbc3-4f6f-8b30-21649d1fc105)
I only seem to have these options.

However I checked via my web browser, instead of the Jellyfin Windows App, and there the setting is.
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/e684397d-6407-408e-9709-67b62a188408)

I'll be testing it out after work and update this message, and if it works, I'll also update the original post so people who are facing this issue know how to fix it without going through all the dialogue.
Thank you.

**Comment by joost00719** [comment]
Just tested it with the "Burn subtitles" option set to "All".
After that I rebooted the chrome cast, rebooted Jellyfin server and tried again.
Unfortunately, it did not work. It doesn't seem like the subtitles are getting burned in correctly, even though the menu suggests it's global for that user, and not for the client since it doesn't say "These settings affect subtitles on this device" on that part.

I tried to test it by lowering the quality to the lowest setting possible and see if it affected the quality. In the browser it works, and ffmpeg is being shown in intel_gpu_top utility, in the Windows app and in the AndroidTv app it did not work and it is a direct play.

Is there any other way I can force subtitle burn in and/or transcoding without sacraficing bitrate/quality?

**Comment by hcharbonnier** [comment]
I don't really know, each time I modify this setting it take times before it became effective, and I don't konw why.

**Comment by joost00719** [comment]
> I don't really know, each time I modify this setting it take times before it became effective, and I don't konw why.

I waited 2 days, and it did not become effective.

**Comment by joost00719** [comment]
![image](https://github.com/jellyfin/jellyfin-androidtv/assets/21021559/71fd2f3e-608a-4036-9993-c74f7171eb1a)
Just checked in the JF discord server, and according to "EUPHIE IS SOOOO CUUUUUUUTE" (nice name btw), the option is only client-sided, which explains why the option does not do anything on the AndroidTV app.

https://discord.com/channels/772232779534172171/772247115681890314/1232034632296562839

**Comment by pierfied** [comment]
I've been digging into this a little bit and it would seem that the problem is due to exoplayer returning an incorrect value for the position when jellyfin calls `getCurrentPosition()`. This would explain why the subtitles are shown at the right time relative to the time according to the player but the video appears to be out of sync.  It's possible this could be related to the following exoplayer issue: https://github.com/androidx/media/issues/356

**Comment by joost00719** [comment]
@pierfied that looks exactly like the issue I'm facing. Thanks for figuring out that it's actually unrelated to JF.

Would it be possible to get an option in the androidTV app menu to burn in subtitles? Maybe it's not the "technically correct" fix, but that would fix the sync issue.

**Comment by pierfied** [comment]
@joost00719  this is actually my first time ever looking at the JF codebase as I just wanted to try to dig into this exact issue. So, unfortunately, I don't think I'm the right person to tackle adding an option for burning in subtitles, but I'm sure another developer here may be able to.

However, I have been playing a bit with figuring out how to get around this particular issue. As far as I can tell, using the HLS protocol (as JF + exoplayer natively tries to do) results in exoplayer incorrectly adjusting the timestamps due to to how the JF server is returning TS segments. The JF client is requesting 6 second segments, but the returned segments contain closer to 10 seconds worth of frames for reasons that are beyond my level of knowledge (I think it has something to do with wanting to start each segment with a keyframe). As far as I can tell, exoplayer assumes that the first frame in the segment should correspond to the beginning of the 6 second segment even though that may not actually be the case. The problem then occurs when exoplayer adjusts all timestamps to force that first frame to fall on the beginning of the 6 second segment, resulting in incorrect timestamps for each frame.

Now, I've been trying some stuff to get around this issue and one workaround I've been playing with is simply disabling HLS playback with exoplayer and using a similar transcoding profile as the libVLC-based player:
https://github.com/jellyfin/jellyfin-androidtv/compare/master...pierfied:jellyfin-androidtv:disable-hls

While I don't think this is optimal by any mean, it seems to be working for some limited testing I've done. I'm sure there are some consequences of disabling the HLS protocol that I am unaware of. As I am by no means an expert here and my knowledge of the various protocols, containers, and codecs is very limited, I would greatly appreciate any insights here from more experienced developers on this one.

**Comment by o-pteron** [comment]
I have exactly the same issue

**Comment by sotiris-bos** [comment]
I also have the same issue. And using libvlc causes micro-stuttering making jellyfin almost unusable on Android TV.

Maybe add some "Disable HLS", "Force Burn subtitles" options on the Android TV client app as a quick workaround?

**Comment by aculotarpa** [comment]
I have this very same problem not only when I stop and restart a video, but also when I jump back and forward during playback. The subtitles timing restart from zero.
v. 0.16.11

**Comment by pierfied** [comment]
It seems like this particular issue also shows up across other JF clients and I think this is a server-side issue with how HLS is handled. Disabling HLS in the android tv client has worked well for me for a while now. I think adding an option in the settings to disable HLS playback might be a good solution until the HLS timestamp problem is properly diagnosed/fixed.

**Comment by viown** [comment]
> It seems like this particular issue also shows up across other JF clients and I think this is a server-side issue with how HLS is handled. Disabling HLS in the android tv client has worked well for me for a while now. I think adding an option in the settings to disable HLS playback might be a good solution until the HLS timestamp problem is properly diagnosed/fixed.

@pierfied Hey, just wanted to ask if there are any issues that could arise from disabling HLS. I.e, would transcoding or direct streaming still work?

**Comment by pierfied** [comment]
It looks like this issue may be solved by https://github.com/jellyfin/jellyfin-androidtv/pull/3825.

**Comment by joost00719** [comment]
> It looks like this issue may be solved by #3825.

Unfortunately I cannot test it because I have sold my Chrome Cast.

**Comment by SkyDiver7199** [comment]
Not fixed.
The subtitles work fine in my pc but when I watch same movie on Android TV, it goes out of sync. 
Works ok when I choose to play using external player.
So something is wrong with Jellyfin

**Comment by nielsvanvelzen** [comment]
Did you try with 0.18?

**Comment by naphipps** [comment]
Definitely not fixed on Firefox. There is (and has been) a lot of issues with Jellyfin's support for Firefox. iPhone app works perfect. Debian 12 client works perfect. Fire stick works perfect. Chrome works perfect. Firefox seems to be getting the short stick lately.

**Comment by nielsvanvelzen** [comment]
How do you even use an Android TV app on Firefox??

**Comment by naphipps** [comment]
Are you asking for more information? I don't mind creating a new issue or doing a something myself if we're not going to make any progress on this issue.

**Comment by nielsvanvelzen** [comment]
If you encounter an issue with the **Android TV** app please do. But if it's not, you may want to look in the appropriate repository instead.

I'd like to remind everyone again that this issue is now closed because of changes made for the 0.18 release, which is **not** the current release but an upcoming version.

**Comment by naphipps** [comment]
Thanks. Will stay tuned.

If the issue exists in that release, I'm assuming to follow up here?

**Comment by maru801** [comment]
> Thanks. Will stay tuned.
> 
> If the issue exists in that release, I'm assuming to follow up here?

You need to take this to the relevant github repo. This repo we're currently commenting in is dedicated to the Android TV app. It has nothing to do with Firefox.

The most appropriate repo would be jellyfin-web.

**Comment by naphipps** [comment]
Thanks. I'm assuming yes.

**Comment by lukagouvea** [comment]
I am having the same issue. Here`s what i could learn so far:

Everytime I jump to a timestamp that isn`t loaded yet, the video player wait for the data to be transfered and the subtitle continues to play. Somehow, the subtitle renderer is not in sync with the video renderer in some edge cases. For some time I thought that my subtitle was out of sync, but it only gets out of sync if I jump to an unloaded time of the video. This issue makes impossible to use IntroSkipper in some cases, because it will skip to an unloaded time of the video.

**Comment by ebkalderon** [comment]
Just adding myself to this ticket, the latest version of the app (v0.18.11) still exhibits this issue, even with (what I believe is) direct playing. It seems to be very sensitive to LAN latency as well. The iOS, Android, and Web clients don't suffer from this issue at all.

**Comment by Ifiht** [comment]
Same as all the other hidden off-topic comments. This has been an issue for as long as I've used Jellyfin with anime.
